pulpcore/content/handler.py

```diff
@@ -135,11 +136,11 @@ def _permit(request, distribution):
                     'g': guard.name,
                     'r': str(pe)
                 })
```

Comment:
The docstring needs the exception type updated

Reply:
Ah, good catch.